courses/Module 1/libs/header-attrs-2.8/header-attrs.js

```diff
@@ -0,0 +1,12 @@
+// Pandoc 2.9 adds attributes on both header and div. We remove the former (to
+// be compatible with the behavior of Pandoc < 2.8).
+document.addEventListener('DOMContentLoaded', function(e) {
```

Comment:
I noticed these files got added in the commit. Are they necessary? Maybe we could .gitignore the libs folder?